packages/suite/src/views/wallet/coinmarket/common/CoinmarketTransactions/CoinmarketTransaction/CoinmarketTransactionContainer.tsx

```diff
@@ -0,0 +1,82 @@
+import { variables } from '@trezor/components';
+import { borders, spacingsPx } from '@trezor/theme';
+import styled from 'styled-components';
+
+export const CoinmarketTransactionWrapper = styled.div`
+    display: flex;
+    margin-bottom: ${spacingsPx.lg};
+    border: 1px solid ${({ theme }) => theme.borderDashed};
```

Comment:
I'm not sure I like this, it introduces new visual element. Why not use just Card with fillType "none"?